### DISCUSSION TITLE
Fillter by Upload Date

### DISCUSSION THREAD

**Discussion by WhereAreWeSeeing** [pystardust/ytfzf#208] [Ideas]
Would it be possible to have an option to filter video results by upload date? This could be used in conjunction with the subscriptions feature as well to make it more like it is on the website.

**Comment by pystardust** [comment]
The dates that we get from youtube is of the form `1 day ago` `1 hour ago`. I think the date command can handle this like so `date -d "1 day ago"`.
So yes this would possible. Lets work on it

**Reply by pystardust** [reply]
This has been implemented now

**Reply by SK4G** [reply]
What is the correct way to run the sort by date command? 

I tried `ytfzf -t -c S date -d "1 day ago"` and `ytfzf -t -c S -d "1 day ago"`

**Reply by Euro20179** [reply]
no idea what version of ytfzf you're running, but `yfzf -t -c S --sort` should work.

**Reply by SK4G** [reply]
Was running v2.2-1 non git version from the Arch AUR. Uninstalled and installed the git version from AUR. `ytfzf: git-1390.de16967` 

Non git version subscription scraping was working. Git version returns `Nothing was scraped` with  `ytfzf -t -l -c S --sort`

**Reply by Euro20179** [reply]
You should make a bug report and along  with the other things it asks for you should put the output of `sh -x ytfzf -t -l -c S --sort`
